src/gg.regression.unity.bots/Editor/Scripts/RGSettingsUIRegistrar.cs

```diff
@@ -25,7 +25,7 @@ public static SettingsProvider CreateRGSettingsProvider()
             var provider = new SettingsProvider("Regression Games", SettingsScope.Project)
             {
                 // Create the SettingsProvider and initialize its drawing (IMGUI) function in place:
-                guiHandler = async (searchContext) =>
+                guiHandler = (searchContext) =>
```

Comment:
`async` without `await` was generating a warning in our BossRoom sample, but the message was an error in a partner's project that prevented play.